### ISSUE TITLE
System Message: ERROR en varias páginas

### ISSUE THREAD

**Issue by humitos** [PyAr/pyarweb#230]
* http://beta.python.org.ar/lista/
* http://beta.python.org.ar/wiki/EtiquetaPyAr
* http://beta.python.org.ar/wiki/Proyectos

Entre otras...

```
System Message: ERROR/3 (<string> line 96)
```

**Comment by mgaitan** [comment]
esto es error de markup rst.

es medio agresivo, pero el "line 96" se refiere a la linea 96 al editar la
pagina

On Tue, Mar 10, 2015 at 9:45 PM, Manuel Kaufmann <notifications@github.com>
wrote:

>
>    - http://beta.python.org.ar/lista/
>    - http://beta.python.org.ar/wiki/EtiquetaPyAr
>    - http://beta.python.org.ar/wiki/Proyectos
>
> Entre otras...
>
> System Message: ERROR/3 (<string> line 96)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PyAr/pyarweb/issues/230>.
>



-- 
mgaitan.github.io
textosypretextos.com.ar <http://textosyprextextos.com.ar>

**Comment by mgaitan** [comment]
o sea, no hay "bug" del sitio acá.

**Comment by humitos** [comment]
Claro, pero eso viene de acá: http://python.org.ar/ListaDeCorreo , por ejemplo...

Es un error en la migración de datos de la wiki. ¿Dónde debería reportarlo? Sino, una vez publicada la web van a quedar un sin fin de páginas de wiki mal migradas y que se van a ver muy feas. Lo mínimo que debemos tener es "conocimiento de esos errores" para que, si no queda otra, ir resolviéndolos a mano.

**Comment by mgaitan** [comment]
ok, tenes razón. Armemos un to do list con los fix necesarios de de todas las **paginas wiki importantes**

**Comment by mgaitan** [comment]
actualicé la descripción del ticket

**Comment by alener** [comment]
En http://beta.python.org.ar/wiki/Proyectos  
System Message: ERROR/3 (<string> line 37) id7
Undefined substitution referenced: ";)".

**Comment by galloramiro** [comment]
Hola, 

¿Como están? Retomo este issue y actualizo los errores que quedaron a la vista en la página para que los podamos ir limpiando de a poco. 

Estos son todos los que encontré en la Wiki, si se me paso uno avisen y lo agrego.

- [ ]  [wiki/Eventos](http://www.python.org.ar/wiki/Eventos) 
- `<blockquote id="reuniones">`
- `<h1>System Message: WARNING/2 (&lt;string&gt; line 44) <a href="#id5">id5</a></h1>`
- `<h1>System Message: ERROR/3 (&lt;string&gt; line 9) <a href="#id11">id11</a></h1>`

- [ ]  [wiki/ListaDeCorreos](http://www.python.org.ar/wiki/ListaDeCorreo)
- `<a href="/wiki/MailTopyar-subscribe-AT-python-DOT-org-DOT-ar">[[MailTo(pyar-subscribe AT python DOT org DOT ar)]]</a>`
- `<h1>System Message: WARNING/2 (&lt;string&gt; line 73) <a href="#id2">id2</a></h1>`
- `<h1>System Message: ERROR/3 (&lt;string&gt; line 14) <a href="#id13">id13</a></h1>`

- [ ]  [wiki/Proyectos](http://www.python.org.ar/wiki/Proyectos)
- `<a href="/wiki/Navigationchildren-1">[[Navigation(children, 1)]]</a>`
- `<h1>System Message: ERROR/3 (&lt;string&gt; line 37) <a href="#id7">id7</a></h1>`

- [ ]  [wiki/Recursos](http://www.python.org.ar/wiki/Recursos)
- `<a href="/wiki/MailTomconte-AT-atommica-DOT-com-Martin-Conte-Mac-Donell">[[MailTo(mconte AT atommica DOT com, Martín Conte Mac Donell)]]</a>`
- `<td>Por orden alfabético.</td>`

- [ ]  [wiki/Charlas](http://www.python.org.ar/wiki/Charlas)
- `<h1>System Message: WARNING/2 (&lt;string&gt; line 396)</h1>`

- [ ]  [wiki/TareasPendientes](http://www.python.org.ar/wiki/TareasPendientes)
- `<p><a href="/wiki/Navigationchildren">[[Navigation(children)]]</a></p>`
- `<div id="id3">`

- [ ]  [wiki/PreguntasFrecuentes](http://www.python.org.ar/wiki/PreguntasFrecuentes)
- `<div id="id1">`

- [ ]  [wiki/Trabajos](http://www.python.org.ar/wiki/Trabajos)
- `<h1>System Message: WARNING/2 (&lt;string&gt; line 1081) <a href="#id3">id3</a></h1>` *N

- [ ]  [wiki/Donaciones](http://www.python.org.ar/wiki/Donaciones)
- `<a href="/wiki/DineroMail">[[DineroMail()]]</a>`
- `<a href="/wiki/CuentaDigital">[[CuentaDigital()]]</a>`

- [ ]  [wiki/AdoptaUnNewbie](http://www.python.org.ar/wiki/AdoptaUnNewbie)
- `<div id="id10">`
